StatePrinter.Tests/IntegrationTests/DictionaryTest.cs

```diff
@@ -803,13 +836,13 @@ public void Dictionary_complexKeySimpleValue()
                 var expected = @"<Root type='Dictionary(Person, Int32)'>
     <Element type='KeyValuePair(Person, Int32)'>
         <key type='Person'>
-            <Name>""Douglas""</Name>
+            <Name>Douglas</Name>
```

Comment:
I think quotes in the CurlyBraceStyle are useful, especially in single-line mode. Compare these:
  { Address = "Zurich, Switzerland" Name = "Roman" }
vs
  { Address = Zurich, Switzerland Name = Roman }

In XML style quotes aren't necessary because each value is already delimited by XML tags.

Right now the CurlyBraceStyle looks kinda like C# while JSON and XML looks kinda like JSON and XML respectively. That's nice.

By the way, there is one thing I forgot to mention, which is related. JSON formatter used to produce strings like this:

  {42: 1337}

This isn't legal JSON -- object fields can't be numbers. I changed it to this:

  {"42": "1337"}